support-workers/build.sbt

```diff
@@ -46,10 +46,11 @@ riffRaffPackageType := assembly.value
 riffRaffManifestProjectName := s"support:support-workers-mono"
 riffRaffManifestBranch := Option(System.getenv("BRANCH_NAME")).getOrElse("unknown_branch")
 riffRaffBuildIdentifier := Option(System.getenv("BUILD_NUMBER")).getOrElse("DEV")
-riffRaffManifestVcsUrl := "git@github.com/guardian/support-workers.git"
+riffRaffManifestVcsUrl := "git@github.com/guardian/support-frontend.git"
 riffRaffUploadArtifactBucket := Option("riffraff-artifact")
 riffRaffUploadManifestBucket := Option("riffraff-builds")
 riffRaffArtifactResources += (file("support-workers/cloud-formation/target/cfn.yaml"), "cfn/cfn.yaml")
+riffRaffArtifactResources += (file("support-workers/target/scala-2.12/support-workers-it.jar"), "it-tests/support-workers-it.jar")
```

Comment:
the artifact is built with "it:assembly" and rather than riffraff artifact plugin find it automatically, we hard code the jar and pick it up as a side artifact during the main upload and deploy.
Another option would be to have a `IntegrationTest / riffraff.yaml` and all the rest, and upload and deploy the IT separately from the rest of the workers, effectively as a separate project.
However it didn't look like riffraff artifact plugin supports alternative names for the yaml file so I would have to add that parameter first.